project/plugins.sbt

```diff
@@ -19,7 +19,8 @@ addSbtPlugin("net.virtual-void" % "sbt-dependency-graph" % "0.9.0") // for advan
 addSbtPlugin("io.spray" % "sbt-boilerplate" % "0.6.1")
 addSbtPlugin("org.foundweekends" % "sbt-bintray" % "0.5.1")
 addSbtPlugin("com.lightbend.sbt" % "sbt-javaagent" % "0.1.4")
-addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "0.4")
-addSbtPlugin("com.lightbend" % "sbt-whitesource" % "0.1.6")
+addSbtPlugin("com.lightbend.akka" % "sbt-paradox-akka" % "0.5")
```

Comment:
Will that break Java / Scala switching for now, @raboof?

Reply:
Oh, well-spotted - yes it will, 0.5 assumes switching can be done in all-javascript and we're not quite there yet. Probably best to stick to 0.4 until we have completed the consolidation effort

Reply:
Reverted that particular change.